vwr/driver.py

```diff
@@ -14,6 +15,10 @@ class CirculatingBath(object):
 
     This class communicates with the circulating bath over UDP sockets.
     """
+    max_delay = 60
+    initial_delay = 5
```

Comment:
Doing the math here, the reconnects are 5, 15, 45, 60, 60, 60. That doesn't seem ideal.

If `initial_delay` was 1 and `factor` was 2, then it'd be 1, 2, 4, 8, 16, 32, 60, 60, 60. That'd have better behavior for temporary disconnects, while still dropping to a once-per-minute check reasonably quickly.